subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/tasks/CreateDependencyInfoFile.java

```diff
@@ -1,74 +1,76 @@
 package org.shipkit.internal.gradle.java.tasks;
 
 import org.gradle.api.artifacts.*;
-import org.json.simple.JsonArray;
-import org.json.simple.JsonObject;
-import org.json.simple.Jsoner;
+import org.shipkit.internal.gradle.util.StringUtil;
 import org.shipkit.internal.notes.util.IOUtil;
 
-import java.util.ArrayList;
-import java.util.LinkedHashMap;
+import java.util.SortedSet;
+import java.util.TreeSet;
 
 public class CreateDependencyInfoFile {
 
-    private static final String DESCRIPTION = "This file was generated by Shipkit Gradle plugin. See http://shipkit.org.";
+    public static final String SEPARATOR = ":";
+    public static final String NEWLINE = "\r\n";
+    public static final String DEPENDENCY_INDENT = NEWLINE + " - ";
+    public static final String ARTIFACT_INDENT = NEWLINE + "   - ";
 
-    public void createDependencyInfoFile(CreateDependencyInfoFileTask task) {
-        JsonObject result = createJsonObject();
-        result.put("description", DESCRIPTION);
+    private static final String DESCRIPTION = "This file was generated by Shipkit Gradle plugin. " +
+        "It contains all declared dependencies of the project. See http://shipkit.org." + NEWLINE +
+        "The format of dependencies is: group:name:version" + NEWLINE +
+        "Each of the dependencies may contain artifacts (deeper nested) formatted like: artifactName:classifier:type:extension";
 
-        JsonArray dependencies = createJsonArray();
+    public void createDependencyInfoFile(CreateDependencyInfoFileTask task) {
+        String result = "# Description" + NEWLINE
+            + DESCRIPTION + NEWLINE + NEWLINE
+            + "# Dependencies";
 
+        //sorting dependencies to assure that they are always in the same order
+        //without depending on Gradle implementation
+        SortedSet<String> dependencies = new TreeSet<String>();
         for (Dependency dependency: task.getConfiguration().getAllDependencies()) {
             if (dependency instanceof ModuleDependency) {
-                addResolvedDependency(dependencies, (ModuleDependency) dependency);
+                String dep = getResolvedDependency(task, (ModuleDependency) dependency);
+                dependencies.add(dep);
             }
         }
 
-        result.put("dependencies", dependencies);
+        result += DEPENDENCY_INDENT + StringUtil.join(dependencies, DEPENDENCY_INDENT);
 
-        IOUtil.writeFile(task.getOutputFile(), Jsoner.prettyPrint(result.toJson()));
+        IOUtil.writeFile(task.getOutputFile(), result.toString());
     }
 
-    private void addResolvedDependency(JsonArray dependencies, ModuleDependency dependency) {
+    private String getResolvedDependency(CreateDependencyInfoFileTask task, ModuleDependency dependency) {
```

Comment:
I suggest to rename this method because we no longer work on resolved dependencies.